main.go

```diff
@@ -21,7 +21,7 @@ var (
 	sequinsVersion string
 
 	bind       = kingpin.Flag("bind", "Address to bind to. Overrides the config option of the same name.").Short('b').PlaceHolder("ADDRESS").String()
-	root       = kingpin.Flag("root", "Where the sequencefiles are. Overrides the config option of the same name.").Short('r').PlaceHolder("URI").String()
+	source     = kingpin.Flag("source", "Where the sequencefiles are. Overrides the config option of the same name.").Short('r').PlaceHolder("URI").String()
```

Comment:
slightly concerned about making this hard flip because this will break existing deployments using `--root` .. maybe deprecate `--root` as a transition mechanism?